kubernetes/aks/system/vault/helm/values.yaml

```diff
@@ -1153,26 +1144,27 @@ serverTelemetry:
 
     # Deploy the PrometheusRule custom resource for AlertManager based alerts.
     # Requires that AlertManager is properly deployed.
-    enabled: false
+    enabled: true
 
     # Selector labels to add to the PrometheusRules.
     # When empty, defaults to:
     #  release: prometheus
-    selectors: {}
+    selectors:
+      release: kube-prometheus-stack
 
     # Some example rules.
-    rules: []
-    #  - alert: vault-HighResponseTime
-    #    annotations:
-    #      message: The response time of Vault is over 500ms on average over the last 5 minutes.
-    #    expr: vault_core_handle_request{quantile="0.5", namespace="mynamespace"} > 500
-    #    for: 5m
-    #    labels:
-    #      severity: warning
-    #  - alert: vault-HighResponseTime
-    #    annotations:
-    #      message: The response time of Vault is over 1s on average over the last 5 minutes.
-    #    expr: vault_core_handle_request{quantile="0.5", namespace="mynamespace"} > 1000
-    #    for: 5m
-    #    labels:
-    #      severity: critical
+    rules:
```

Comment:
two alerts with same name is a good practice? won't this double up alerts for anything over 1s?